site/content/kapp/docs/latest/apply.md

```diff
@@ -84,6 +84,24 @@ Especially useful when added to Jobs. For example, see [examples/resource-orderi
 
 Possible values: "" (default), `containerName1`, `containerName1,containerName2`
 
+### kapp.k14s.io/exists (Available in v0.43.0+)
+
+`kapp.k14s.io/exists` allows us to wait for non-kapp owned resources. 
+
+If the resource is not present already, then kapp uses the `exists` operation for such resources and waits until the resource is created by the cluster (or some external agency).
+
+If the resource is already present, then `noop` is used for it.
+
+Possible values: "".
+
+### kapp.k14s.io/noop (Available in v0.43.0+)
+
+`kapp.k14s.io/noop` allows kapp to know about non-kapp owned resources. 
+
+kapp always uses the `noop` operation for these resources.
```

Comment:
Add example usage? 
CRDs not owned by kapp for an instance

Reply:
Should we add the e2e test link as an example?

Reply:
I would rather add one line describing. the use case here without the YAML. If we wanna add the YAML we should use the shortcode Renu used

Reply:
> I would rather add one line describing. the use case here
Gotcha. Let me think of a good use case.

Reply:
@praveenrewar please add an example